### PR TITLE
miscellaneous changes

### PR DESCRIPTION
**Pull request by wd15**
Lots of annoyingly small changes

Address #1051, #1046, #1045, #1041, #1044, #1049, #937

These changes can be [viewed live][live-site].

**Remember to tear down the [live site][live-site] when merging
or closing this pull request.**

[live-site]: https://random-cat-1055.surge.sh

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/pfhub/1055)

### CODE REVIEW

File: _data/codes.yaml [hunk 1]
```diff
@@ -146,3 +146,19 @@
...
+- name: Sfepy
+  uploads: true
+  logo: http://sfepy.org/doc-devel/_static/sfepy_logo_title_small.png
+  home_page: http://sfepy.org
+  description: >-
+    SfePy is a software for solving systems of coupled partial
+    differential equations (PDEs) by the finite element method in 1D,
+    2D and 3D.
+  badges:
+    - href: http://www.openphase.de/license/index.html
```
Comment (tkphd) [inline _data/codes.yaml]: Wrong link for SfePy.

File: wiki/publications.md [hunk 2]
```diff
@@ -32,16 +32,46 @@ article in *Computational Materials Science* entitled ["Phase field
...
+Jokisaari][aj], [Daniel Schwen][ds], [Jon Guyer][jg], [Larry
+Aagesen][la], [Olle Heinonen][oh], [Michael Tonks][mt], [Peter
+Voorhees][pv] and [James Warren][jw] published a peer-reviewed journal
+article in *Journal of open research software* entitled ["PFHub: The
+Phase-Field Community
+Hub"](http://dx.doi.org/10.1016/j.commatsci.2016.09.022){:target="_blank"}.
+
+<iframe width="900"
+        height="700"
+        src="https://drive.google.com/file/d/1iJvZzaxAQjNYpDkl20Uyb33HbDKfys8x/preview"
```
Comment (tkphd) [inline wiki/publications.md]: As *JORS* is open-access, the DOI link (https://doi.org/10.5334/jors.276) would be preferable to this bootleg copy.

File: _data/codes.yaml (hunk 1 above)
Comment (wd15) [reply _data/codes.yaml]: Fixed

File: wiki/publications.md (hunk 2 above)
Comment (wd15) [reply wiki/publications.md]: Doesn't work unfortunately. Can't be embedded and is not a PDF view either. JORS also doesn't have a PDF viewer so if I want to embed the view then I need another place to host the PDF. Also, no such thing as a bootleg copy with [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).

**Pull request merged**